docs/install-bitbucket-server.md

```diff
@@ -42,6 +42,13 @@
                 - name: bitbucket
             inputResources:
                 - name: cron_trigger
+            # Set the relevant language based on your project, read more: https://www.jfrog.com/confluence/display/JFROG/Choosing+your+Runtime+Image#ChoosingyourRuntimeImage-ChoosingLanguageandVersion
+            # runtime:
+            #   type: image
+            #   image:
+            #     auto:
+            #       language: 
+            #       versions: 
```

Comment:
Was the above section added by mistake? What does it mean? The line saren't aligned.